app/domain/authentication/authn_gcp/authenticator.rb

```diff
@@ -41,6 +59,7 @@ class Authenticator
       # `Call` objects created each time `call` is run.
       def valid?(input)
         call(authenticator_input: input)
+        true
```

Comment:
by Ruby convention, methods that ends with `?` needs to return boolean.
Since we do not do it, we violate this convention.
Still, we do implicitly do it, because Ruby returns the result of the last statement.
Can you guess what it is?


It is `logger.debug()`. Which, in the logger we use, return `true`...

Instead of all this mess, we better be explicit to return true, since `valid?` will fail with raised error and not by returning `false`.

Reply:
To make it clearer define a var `valid = true` and return it.

Reply:
IMO it is a clear ruby syntax.
It might be needed in other languages, but in ruby, the it is clear that
```
def is_valid?
  true
end
```
means that it is valid.

Reply:
i agree with @abrahamko on this

Reply:
In a single line function it is more clear.